### PR TITLE
Ekoparty Security Conference. ASVS and MSTG talk in Spanish

### PR DESCRIPTION
**Pull request by martinmarsicano**
"OWASP Mobile Project and how to use it for white hat hacking" slides for the talk given at Ekoparty Security Conference.

### CODE REVIEW

**Comment by martinmarsicano** [comment]
Just slides for now, whenever the video is published by the event organizers, I will link that too.

**Comment by martinmarsicano** [comment]
@cpholguera can you review this?

File: index.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ A checklist is available for use in security assessments that is based on the MA
 
 Below you can find a list of upcoming and previous talks:
 
+- Ekoparty Security Conference, September 2020 - (Spanish) OWASP Mobile Project and how to use it for white hat hacking: [recording](https://www.youtube.com/watch?v=Son1elZ_0bw), [Slides](https://docs.google.com/presentation/d/1DLN5osuMjTi4hpanipCglUxYkH_gvitNyJjuv6s7SRk/edit?usp=sharing)
```
Comment (cpholguera) [inline index.md:74]:
```suggestion
- Ekoparty Security Conference, September 2020 - (Spanish) OWASP Mobile Project and how to use it for white hat hacking: [Recording](https://www.youtube.com/watch?v=Son1elZ_0bw), [Slides](https://docs.google.com/presentation/d/1DLN5osuMjTi4hpanipCglUxYkH_gvitNyJjuv6s7SRk/edit?usp=sharing) 
```

**Review by cpholguera — APPROVED** [review]
Thanks @martinmarsicano !

**Pull request merged**